docs/en/cookbook/advanced-field-value-conversion-using-custom-mapping-types.rst

```diff
@@ -13,7 +13,7 @@ class, converting the value on the database-level or a combination of both.
 This article describes the third way by implementing the MySQL specific column
 type `Point <http://dev.mysql.com/doc/refman/5.5/en/gis-class-point.html>`_.
 
-The ``Point`` type is part of the `Spatial extension <http://dev.mysql.com/doc/refman/5.5/en/spatial-extensions.html>`_
+The ``Point`` type is part of the `Spatial eemtension <http://dev.mysql.com/doc/refman/5.5/en/spatial-eemtensions.html>`_
```

Comment:
Good catch!

Reply:
Honestly absolutely no idea how this got here 🤔, will revert.